src/coreclr/src/ilasm/grammar_after.cpp

```diff
@@ -924,10 +924,6 @@ int yylex()
                                     tok = parse_literal(curSym, curPos, FALSE);
                                     if(tok == QSTRING)
                                     {
-                                        // insert UTF-8 BOM prefix
```

Comment:
Will strings encoded in UTF-8 still work?

Reply:
I was not able to figure out what this was supposed to help with. Non-ASCII QSTRINGs (quoted strings) never worked in this position as far as I can tell. The following:

```
#define NAME "ššš"

.assembly NAME
```

fails in all .NET Framework and earlier .NET Core versions of ilasm. The case that this change is fixing is:

```
#define NAME "sss"

.assembly NAME
```

Reply:
Ok, thanks for the details.